static-site/posts/updates/2019.3.4-update.udon

```diff
@@ -0,0 +1,52 @@
+:-  :~  date/'~2019.3.14'
+        title/'~2019.3.4 Update'
+        preview/'A short update from the Tlon team for the week of ~2019.3.4.'
+        author/'~hidrel-fabtel'
+        type/'post'
+    ==
+;>
+
+# ~2019.3.4 Update
+
+Here’s a short update from the team for the week of `~2019.3.4`:
+
+;img@"https://media.urbit.org/site/posts/updates/~2019.3.4-update-1.jpg"(width "100%");
+
+## Network
+
+We breached successfully. You can read about [what we fixed](https://github.com/urbit/arvo/pull/1101). Issues ocassionally crop up as people join the network, but it looks to be smooth sailing as we make our way toward [cc-release](https://github.com/urbit/urbit/tree/cc-release) and new ames.
```

Comment:
Our convention is to either capitalize Ames or write it as `%ames`. So just pick one of those and be consistent with it.